app/section/sections/browser-additional-information/ui.test.js

```diff
@@ -0,0 +1,181 @@
+  await pageSetup(t);
+
+  nock('http://localhost:8080')
+    .put('/api/v1/Solutions/S100000-001/sections/browser-additional-information')
+    .reply(400, {
+      maxLength: ['additional-information'],
+    });
+
+  const oneHundredCharacters = '0123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789';
+  const thousandCharacters = oneHundredCharacters.repeat(10);
+
+  const errorSummary = Selector('[data-test-id="error-summary"]');
+  const errorSummaryList = Selector('.nhsuk-error-summary__list');
+  const additionalInformation = Selector('[data-test-id="question-additional-information"]');
+  
+  const submitButton = Selector('[data-test-id="section-submit-button"] button');
+  
+  await t
+  .expect(errorSummary.exists).notOk()
+  .typeText(additionalInformation, `${thousandCharacters}0`, { paste: true })
+  .click(submitButton)
+  .expect(errorSummary.exists).ok()
+  .expect(errorSummaryList.find('li').count).eql(1)
+  .expect(errorSummaryList.find('li:nth-child(1)').innerText).eql('Additional information is over the character limit')
+  .expect(errorSummaryList.find('li:nth-child(1) a').getAttribute('href')).eql('#additional-information')
+  .expect(additionalInformation.find('.nhsuk-textarea--error').exists).ok()
+  .expect(additionalInformation.find('.nhsuk-error-message').innerText).eql('Error:\nAdditional information is over the character limit')
+});
+
+test('should goto anchor when clicking the additional information required error link', async (t) => {
+  await pageSetup(t);
+
+  nock('http://localhost:8080')
+    .put('/api/v1/Solutions/S100000-001/sections/browser-additional-information')
+    .reply(400, {
+      maxLength: ['additional-information'],
+    });
+
+  const oneHundredCharacters = '0123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789';
+  const thousandCharacters = oneHundredCharacters.repeat(10);
+
+  const errorSummary = Selector('[data-test-id="error-summary"]');
+  const errorSummaryList = Selector('.nhsuk-error-summary__list');
+  const additionalInformation = Selector('[data-test-id="question-additional-information"]');
+  
+  const submitButton = Selector('[data-test-id="section-submit-button"] button');
+
+  const getLocation = ClientFunction(() => document.location.href);
+
+  await t
+    .expect(errorSummary.exists).notOk()
+    .typeText(additionalInformation, `${thousandCharacters}0`, { paste: true })
+    .click(submitButton)
+    .expect(errorSummary.exists).ok()
+    .expect(errorSummaryList.find('li:nth-child(1) a').count).eql(1)
+    .expect(errorSummaryList.find('li:nth-child(1) a').getAttribute('href')).eql('#additional-information')
+    .click(errorSummaryList.find('li:nth-child(1) a'))
+    .expect(getLocation()).contains('/S100000-001/section/browser-additional-information#additional-information');
+});
+

```

Comment:
I would add another test that makes sure it goes back to the brower-based dashboard on a successful submit.

we've done that for the other sections that are browser-based subSections. Something like below ->
```
test('should goto the browser based dashboard when clicking the submit button', async (t) => {
  await pageSetup(t, 200, connectivityAndResolutionMarketingData);

  nock('http://localhost:8080')
    .put('/api/v1/Solutions/S100000-001/sections/connectivity-and-resolution')
    .reply(200, connectivityAndResolutionMarketingData);

  nock('http://localhost:8080')
    .get('/api/v1/Solutions/S100000-001/sections/browser-based')
    .reply(200, {});

  const submitButton = Selector('[data-test-id="section-submit-button"] button');
  const getLocation = ClientFunction(() => document.location.href);

  await t
    .expect(submitButton.exists).ok()
    .click(submitButton)
    .expect(getLocation()).contains('/S100000-001/dashboard/browser-based');
});
```